src/components/TodosList.jsx

```diff
@@ -0,0 +1,19 @@
+/* eslint-disable react/prop-types */
+import TodoItem from './InputTodo';
+
```

Comment:
Please import the right component to be able to correctly display your todo items.

Reply:
Thanks @serengia, I will make sure to add these changes ASAP.